mifosng-android/src/main/java/com/mifos/mifosxdroid/online/search/SearchFragment.java

```diff
@@ -133,6 +133,7 @@ public void showSearchedResources(List<SearchedEntity> searchedEntities) {
     public void showNoResultFound() {
         showAlertDialog(getString(R.string.dialog_message),
                 getString(R.string.no_search_result_found));
+        rv_search.setAdapter(null);
```

Comment:
setting adapter null in searchView sometimes leads to unexpected crashes. Instead of this, you can set resultSet array of adapter to an empty list like this `searchAdapter.setSearchResults(Collections.<SearchedEntity>emptyList());` .